CONTRIBUTING.md

```diff
@@ -14,8 +14,67 @@ search the list to see if your issue has already been raised.
 A good bug report is one that make it easy for us to understand what you were
```

Comment:
A good bug report is one that make it easy => A good bug report is one that makes it easy